lab-ron/app/component/app-gallery-item/app-gallery-item-directive.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const angular = require('angular');
+angular.module('demoApp')
+.directive('appGalleryItem', function(){
+  return {
+    restrict: 'E',
+    replace: true,
+    scope: {
+      imgSrc: '@',
```

Comment:
Here you are using '@', but in some of your other directives, you're using '=' for the same properties. Why is that?